extension/src/content/restyling.js

```diff
@@ -0,0 +1,149 @@
+import $ from 'jquery';
```

Comment:
github diff doesn't pick up line changes in this file for some reason... were there significant changes in this one?

Reply:
nope, just imports